README.md

```diff
@@ -18,7 +18,7 @@ use ZipkinTracer::RackHandler, config # config is optional
 where `Rails.config.zipkin_tracer` or `config` is a hash that can contain the following keys:
 
 * `:service_name` **REQUIRED** - the name of the service being traced. There are two ways to configure this value. Either write the service name in the config file or set the "DOMAIN" environment variable (e.g. 'test-service.example.com' or 'test-service'). The environment variable takes precedence over the config file value.
-* `:service_port` **REQUIRED** - the port of the service being traced (e.g. 80 or 443)
+* `:service_port` - the port of the service being traced (e.g. 80 or 443)
```

Comment:
can't we just delete this?

Reply:
updated in https://github.com/openzipkin/zipkin-ruby/pull/139/commits/29af492d4f5a0f134b7799aae196a5b8b8ba961c